csharp/src/Drivers/BigQuery/BigQueryStatement.cs

```diff
@@ -75,6 +76,24 @@ public override QueryResult ExecuteQuery()
             readClientBuilder.Credential = this.credential;
             BigQueryReadClient readClient = readClientBuilder.Build();
 
+            if (results.TableReference == null)
+            {
+                ListJobsOptions listJobsOptions = new ListJobsOptions();
+                listJobsOptions.ParentJobId = results.JobReference.JobId;
+                PagedEnumerable<JobList, BigQueryJob> joblist = client.ListJobs(listJobsOptions);
+                BigQueryJob firstQueryJob = new BigQueryJob(client, job.Resource);
+                foreach (BigQueryJob childJob in joblist)
+                {
+                    var tempJob = client.GetJob(childJob.Reference.JobId);
+                    var query = tempJob.Resource?.Configuration?.Query;
+                    if (query != null && query.DestinationTable != null && query.DestinationTable.ProjectId != null && query.DestinationTable.DatasetId != null && query.DestinationTable.TableId != null)
```

Comment:
What would it mean if there were multiple child jobs that match these criteria? Should that produce an error?

Reply:
When the query is a stored procedure or multiple statements, the null problem will be triggered stably. When there are multiple child query jobs, the result of the first query will be returned.

Reply:
Won't the current implementation return the last child job instead of the first one? Do we need a `break;` after the assignment to `firstQueryJob`?